File: ISP/HW/hw6/ContourData.py
import numpy as np
import matplotlib.pyplot as plt

array = np.loadtxt('/Users/JCHAO/Documents/UT/YEAR6/ISP/HW/hw6/contourData.dat.txt')
plt.pcolormesh(array, cmap = 'YlOrRd_r')
plt.colorbar()
plt.title('Contour Data')
plt.xlabel("X")
plt.ylabel("Y")
plt.show()

File: ISP/HW/hw6/ContourData.r

data <- matrix(scan("/Users/JCHAO/Documents/UT/YEAR6/ISP/HW/hw6/contourData.dat.txt"), nrow=1000, byrow=TRUE)
colfunc <- colorRampPalette(c("red", "yellow")) 
filled.contour(data, col=colfunc(20), plot.title = title(main='Contour Data', xlab='x',ylab='y'))
